src/pages/settings/PasswordPage.js

```diff
@@ -152,77 +152,81 @@ class PasswordPage extends Component {
+                                            secureTextEntry
+                                            autoCompleteType="password"
+                                            textContentType="password"
+                                            value={this.state.newPassword}
+                                            hasError={this.state.errors.newPassword || this.state.errors.newPasswordSameAsOld}
+                                            errorText={this.state.errors.newPasswordSameAsOld
+                                                ? this.getErrorText('newPasswordSameAsOld')
+                                                : this.getErrorText('newPassword')}
+                                            onChangeText={text => this.clearErrorAndSetValue('newPassword', text, ['newPasswordSameAsOld'])}
+                                            onSubmitEditing={this.submit}
+                                        />
+                                        {shouldShowNewPasswordPrompt && (
+                                        <Text
+                                            style={[
+                                                styles.textLabelSupporting,
+                                                styles.mt1,
+                                            ]}
+                                        >
+                                            {this.props.translate('passwordPage.newPasswordPrompt')}
+                                        </Text>
+                                        )}
+                                    </View>
+                                    {_.every(this.state.errors, error => !error) && !_.isEmpty(this.props.account.error) && (
+                                    <Text style={styles.formError}>
+                                        {this.props.account.error}
+                                    </Text>
+                                    )}
+                                </ScrollView>
+                                <FixedFooter style={[styles.flexGrow0]}>
+                                    <Button
+                                        success
+                                        isLoading={this.props.account.loading}
+                                        text={this.props.translate('common.save')}
+                                        onPress={this.submit}
+                                    />
+                                </FixedFooter>
+                            </>
                         )}
-                    </ScrollView>
-                    <FixedFooter style={[styles.flexGrow0]}>
-                        <Button
-                            success
-                            isLoading={this.props.account.loading}
-                            text={this.props.translate('common.save')}
-                            onPress={this.submit}
-                        />
-                    </FixedFooter>
                 </KeyboardAvoidingView>
```

Comment:
Too many changes going on here, since it involves two screens. One screen is created as a component, one is left out here, Shouldn’t move it to a separate component? 

cc: @chiragsalian

Reply:
Well this [block](https://github.com/Expensify/App/blob/636b43ceb28437fb18d2bb18887d37077c23ca5f/src/pages/settings/PasswordPage.js#L163-L227) is pretty much unchanged from sahil so i don't feel strongly about asking him to change it. 

Plus i'm not a big fan of "refactors", or updating code unrelated to the changes since its easy to loose time on it for little benefit. So unless you feel strongly i think its okay to leave this as is for now. If someone in the future is working on updating this section then i think its a good time to tell them to move it to a component if needed.

Reply:
Looks good for now!